src/Query.php

```diff
@@ -700,6 +700,117 @@ protected function _render_set_values()
     }
     /// }}}
 
+    // {{{ Limit 
+    /**
+     * Limit how many rows will be returned.
+     *
+     * @param int $cnt   Number of rows to return
+     * @param int $shift Offset, how many rows to skip
+     *
+     * @return DB_dsql $this
+     */
+    public function limit($cnt, $shift = null)
```

Comment:
In SQL it's `limit [offset,]row_count`, but in method parameters it's in opposite order - `cnt, shift`. This can be a bit hard to get used to and not mix these parameters.